backend/base.py

```diff
@@ -857,3 +863,15 @@ def getUserRegisteredEvents():
         response = {"status": "Error", "message": str(e)}
         statusCode = 500
     return jsonify(response),statusCode
+
+def calculate_tdee(height,weight,age,gender,activityLevel):
+    kg_weight = float(weight)*0.45359237
+    cm_height = float(height)*30.48
+    common_bmr = (10*kg_weight) + (6.25*cm_height) - (5*int(age))
+    if gender == "Male":
+        bmr = common_bmr + 5
+    else:
+        bmr = common_bmr - 161
+    pal = {'Minimal': 1.2,'Light': 1.375, 'Moderate': 1.55, 'Heavy':1.725, 'Athlete': 1.9}
```

Comment:
can we name this something more intuitive? dont understand variable names

Reply:
Have changed it to more intuitive and descriptive names